Added times spent on different parts of assignment

Jonagoo opened the pull request:
Put the times different group members spent on different parts of the assignment into the report.
This should resolve #102

Samuelhertzberg committed:
Entered times

Alexandres, Emils, Franz and Samuels times are now in the table.

Jonagoo committed:
Entered times for Jonathan

Commit:
Fixed table not being displayed correctly

There was something weird with the markdown which led to the table not being displayed properly.

Jonagoo committed:
Updated times

Samuelhertzberg reviewed (approved):
Wow this is amazing, you're such a flitig boye.

Ya boi,
Hacker Hertzberg

Commit:
Added time for Emil and Franz

Franz and Emil did not agree with the effort spent table and therefore adjusted some values.

francislaus reviewed (approved):
Nice work!
